packages/editor-ui/src/components/CredentialCard.test.ts

```diff
@@ -55,4 +61,31 @@ describe('CredentialCard', () => {
 		expect(heading).toHaveTextContent(data.name);
 		expect(badge).toHaveTextContent('John Doe');
 	});
+
+	it('should show Move action only if there is resource permission and not on community plan', async () => {
+		vi.spyOn(settingsStore, 'planName', 'get').mockReturnValue('Enterprise');
+
+		const data = createCredential({
+			scopes: ['credential:move'],
+		});
+		const { getByTestId } = renderComponent({ props: { data } });
+		const cardActions = getByTestId('credential-card-actions');
+
+		expect(cardActions).toBeInTheDocument();
+
+		const cardActionsOpener = within(cardActions).getByRole('button');
+		expect(cardActionsOpener).toBeInTheDocument();
+
+		const controllingId = cardActionsOpener.getAttribute('aria-controls');
+
+		await userEvent.click(cardActions);
+		const actions = document.querySelector(`#${controllingId}`);
+		if (!actions) {
+			throw new Error('Actions menu not found');
+		}
+		await waitFor(() => {
+			expect(actions).toBeInTheDocument();
+		});
```

Comment:
Wouldn't these two actually check the same thing?  

Maybe you could use
```
await waitFor(() => {
  expect(document.querySelector(`#${controllingId}`)).toBeInTheDocument();
});
```

Reply:
It wasn't necessary, I have removed